common/fixation_entry.go

```diff
@@ -996,37 +996,61 @@ func (fs *FixationStore) setVersion(ctx sdk.Context, val uint64) {
 	store.Set(types.KeyPrefix(types.FixationVersionKey), b)
 }
 
-func (fs *FixationStore) Export(ctx sdk.Context) []types.RawMessage {
-	store := prefix.NewStore(
-		ctx.KVStore(fs.storeKey),
-		types.KeyPrefix(fs.prefix))
-	iterator := sdk.KVStorePrefixIterator(store, []byte{})
-	defer iterator.Close()
+func (fs *FixationStore) Export(ctx sdk.Context) types.GenesisState {
+	gs := types.GenesisState{}
 
-	data := []types.RawMessage{}
-	for ; iterator.Valid(); iterator.Next() {
-		data = append(data, types.RawMessage{Key: iterator.Key(), Value: iterator.Value()})
+	gs.Version = fs.getVersion(ctx)
+
+	for _, EntryIndex := range fs.GetAllEntryIndices(ctx) {
```

Comment:
elsewhere the code usually uses "index" (and anyhow no capital).

Reply:
https://github.com/lavanet/lava/pull/678/commits/6ee8bf73cd4a8c56bc341c44c26513105b9f97ec